ouroboros-consensus-test/test-consensus/Test/Consensus/HardFork/Combinator.hs

```diff
@@ -108,6 +110,9 @@ instance Arbitrary TestSetup where
       testSetupEpochSize <- abM $ EpochSize <$> choose (1, 10)
       testSetupK         <- SecurityParam   <$> choose (2, 10)
       -- TODO why does k=1 cause the nodes to only forge in the first epoch?
+
+      -- TODO @js: think about what value should be used here.
+      let testSetupS     = GenesisWindowLength $ 2 * maxRollbacks testSetupK -- TODO
```

Comment:
Is this relation between `maxRollbacks` and `2` assumed to be always this one? If so, do we have an assertion in the code checking this? If not maybe we'd want to vary `s` in the test setup.